zuul.d/edpm_multinode.yaml

```diff
@@ -443,6 +447,10 @@
             network: "172.19.0.0/24"
             gateway: "172.19.0.1"
             vlan: 22
+          external:
+            network: "172.21.0.0/24"
+            gateway: "172.21.0.1"
+            vlan: 44
```

Comment:
this breaks the external (public IPs) connectivity

Reply:
See https://logserver.rdoproject.org/12/53212/15/check/adoption-multinode-to-extracted-crc-no-ceph-novacells/7678a1d/job-output.txt